internal/agents/claude/store.go

```
@@ -286,14 +286,38 @@ func (s Store) legacyInstancePath(instancePath string) (string, bool) {
+	if err != nil {
+		return nil, err
+	}
+	if alias {
+		return roots, nil
+	}
+	return append(roots, legacyRoot), nil
+}
+
 func (s Store) profileInstancePaths(dir string) ([]string, error) {
 	if !safeProfileDir(dir) {
 		return nil, errors.New("Claude profile directory is invalid")
```

golangci-lint (2.13.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)